pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "nerdlandbot"
+version = "0.1.0"
+description = ""
+authors = ["Autom"]
```

Comment:
nitpicking, should be more than only Autom, right ;)

Reply:
It definitely should be, but I don't know what nickname everyone wants there...

Reply:
Would suggest to make it their github usernames?

Reply:
Ah I used the names people used to commit to git, that was a bit easier for me :P People can always change theirs in a PR if they want?